Formula/gdk-pixbuf.rb

```diff
@@ -84,7 +84,17 @@ def module_dir
 
   def post_install
     ENV["GDK_PIXBUF_MODULEDIR"] = "#{module_dir}/loaders"
-    system "#{bin}/gdk-pixbuf-query-loaders", "--update-cache"
+
+    ohai "#{bin}/gdk-pixbuf-query-loaders --update-cache" # Previously handled by `system`.
```

Comment:
I thought it would be a good idea to add that comment in, but I can remove it if needed/desired.